### PR TITLE
Update CLI wording

### PR DESCRIPTION
**Pull request by lucperkins**
A bit of a pass-over lint job for CLI text, for the sake of consistency. Mostly very small nits. This PR introduces _no_ functionality changes and no changes to command or flag names, only to descriptions and help text.

### CODE REVIEW

**Review by bufdev — APPROVED** [review]
I'm going to assume this is a pure improvement and just say ship it.

File: private/buf/cmd/buf/command/beta/registry/commit/commitget/commitget.go [hunk 1]
```diff
@@ -38,7 +38,7 @@ func NewCommand(
 	flags := newFlags()
 	return &appcmd.Command{
 		Use:   name + " <buf.build/owner/repo:ref>",
-		Short: "Get commit details",
+		Short: "Get details about a commit.",
```
Comment (amckinney) [inline private/buf/cmd/buf/command/beta/registry/commit/commitget/commitget.go:41]: nit: From above, we're changing to the `Creates ...` tense, but this is `Get`. Should it be `Gets ...` and `Creates ...` or `Get ...` and `Create ...`?

The same could be said of other commands, like `List ...` vs. `Lists ...`. I kinda like the singular form, but I'm open to your preference as long as it's consistent (I don't know how to refer to these tenses).

File: private/buf/cmd/buf/command/mod/modupdate/modupdate.go [hunk 2]
```diff
@@ -49,11 +49,11 @@ func NewCommand(
 	flags := newFlags()
 	return &appcmd.Command{
 		Use:   name + " <directory>",
-		Short: "Update the modules dependencies. Updates the " + buflock.ExternalConfigFilePath + " file.",
-		Long: "Gets the latest digests for the specified references in the config file, " +
+		Short: "Updates a module's dependencies by updating the " + buflock.ExternalConfigFilePath + " file.",
+		Long: "Fetches the latest digests for the specified references in the config file, " +
```
Comment (amckinney) [inline private/buf/cmd/buf/command/mod/modupdate/modupdate.go:53]: Similar to above (re: `Fetch` vs. `Fetches`).

File: private/buf/cmd/buf/command/registry/registrylogin/registrylogin.go [hunk 3]
```diff
@@ -44,8 +44,8 @@ func NewCommand(
 		// Not documenting the first arg (remote) as this is just for testing for now.
 		// TODO: Update when we have self-hosted.
 		Use:   name,
-		Short: `Login to the Buf Schema Registry.`,
-		Long:  fmt.Sprintf(`This will prompt for your BSR username and a BSR token, and update your %s file with these credentials.`, netrc.Filename),
+		Short: `Log in to the Buf Schema Registry.`,
```
Comment (amckinney) [inline private/buf/cmd/buf/command/registry/registrylogin/registrylogin.go:47]: This one always forces me to look up the correct form 🤦

File: private/buf/cmd/buf/command/registry/registrylogin/registrylogin.go [hunk 4]
```diff
@@ -71,13 +71,13 @@ func (f *flags) Bind(flagSet *pflag.FlagSet) {
 		&f.Username,
 		usernameFlagName,
 		"",
-		"The username to use. By default, this command will prompt for a username.",
+		"The username to use. The command prompts for a username by default.",
```
Comment (amckinney) [inline private/buf/cmd/buf/command/registry/registrylogin/registrylogin.go:74]: `The` vs. `This` compared to the flag description for `--token-stdin` immediately beneath this.

File: private/buf/cmd/buf/command/beta/registry/commit/commitget/commitget.go (hunk 1 above)
Comment (lucperkins) [reply private/buf/cmd/buf/command/beta/registry/commit/commitget/commitget.go:41]: Good call on the discrepancies. My experience with other recent CLI tools suggests that it should be Create, Get, etc. That's not a scientific viewpoint, but let's standardize on that for now.

File: private/buf/cmd/buf/command/registry/registrylogin/registrylogin.go (hunk 3 above)
Comment (lucperkins) [reply private/buf/cmd/buf/command/registry/registrylogin/registrylogin.go:47]: I feel your pain 👊🏼

File: private/buf/cmd/buf/command/registry/registrylogin/registrylogin.go (hunk 4 above)
Comment (lucperkins) [reply private/buf/cmd/buf/command/registry/registrylogin/registrylogin.go:74]: Good catch

**Comment by lucperkins** [comment]
@bufdev Done

**Comment by bufdev** [comment]
Can we merge this?

**Comment by lucperkins** [comment]
@bufdev I'd like to get a definitive thumbs up from @amckinney first

**Review by amckinney — APPROVED** [review]
👏

**Pull request merged**